Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM rust:1.32.0
 
 RUN cargo install cargo-watch
 
+RUN apt-get update
+RUN apt-get install -y ruby-full
```

Comment:
Do we need `ruby-full`? Not that I'm that worried about image size, but we're only executing the tiniest ruby script.

Reply:
no, we don't, i didn't realize there was a `ruby` package